src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.model;
+
+import java.util.Objects;
+
+public class Car {
+
+    public static final int RANDOM_THRESHOLD = 4;
```

Comment:
```suggestion
    private static final int RANDOM_THRESHOLD = 4;
```
접근이 불필요한 상수는 막아주는 것이 좋을 것 같아요 😄